### DISCUSSION TITLE
Parlons photographie ensemble (avec Vimix ! ) Let's talk about photograpy together.

### DISCUSSION THREAD

**Discussion by zcbernard** [brunoherbelin/vimix#135] [Show and tell]
Hello, I use Vimix for Vjing creations, but also for educational purposes in relation to images. I tried to summarize this in this video, it's still in draft form, but your opinions are welcome. 
vidéo : https://youtube.com/playlist?list=PLCbZTUI6KM1i5zu9joqU3D3cmQ_Y4nKx9&feature=shared
bernard

**Comment by brunoherbelin** [comment]
Thanks! but "2 unavailable videos are hidden".. 
Did I miss an instruction or the videos are not public?

**Comment by zcbernard** [comment]
Hello
They are not yet public. This night 00:00

**Comment by zcbernard** [comment]
https://youtu.be/_f0GCQC6xLo

**Comment by zcbernard** [comment]
This is the good link : (just one video)  
https://youtu.be/_f0GCQC6xLo

**Comment by DarrellJonsson** [comment]
Video link is private.

**Comment by zcbernard** [comment]
it will be public this night at 00:00

**Comment by zcbernard** [comment]
Sorry for the setback. I very rarely post on YouTube. I hope to learn... ( ;- ))

**Comment by brunoherbelin** [comment]
Thank you Bernard for sharing your experience with vimix! I find the multi-user interaction with joysticks very creative and the idea of using vimix for working on solutions to creative 'problems' very promising ! 
MERCI !
B.
PS: can I link to your work from the wiki for a page I am planning to call "Made with vimix" ?

**Comment by zcbernard** [comment]
Good evening, thank you for these comments. No problems sharing this work,
it's made for. It would be good to wait a little (2 days), a new version
that I will make in bilingual audio is being prepared. It will be prettier
to look at and hear.
Y'A PAS DE QUOI !
bernard

**Comment by zcbernard** [comment]
And there you have it, latest version of “let’s talk photography together”. It is in French, but by activating the English subtitles, it is very correct. Some errors between play and toy, but it's understandable. You can share as you wish.

https://youtu.be/ue9DrA1dDsI

B
